### PR TITLE
[DRUP-676] APIs landing page theming

### PR DESCRIPTION
**Pull request by minnur**
WIP

### CODE REVIEW

**Comment by shadcn** [comment]
This is ready for QA. I've added the default APIs, specs, images and taxonomy terms. (This also includes style for pagination)

APIs___Apigee

File: composer.json [hunk 1]
```diff
@@ -66,6 +66,9 @@
     "patches": {
       "drupal/core": {
         "`EntityViewBuilder::getBuildDefaults()` doesn't check for `RevisionableInterface`.": "https://www.drupal.org/files/issues/2019-01-10/2951487_15_no-tests.patch"
+      },
+      "drupal/better_exposed_filters": {
+        "Fix issue with #summary_details introduced in Drupal 8.6.x": "https://www.drupal.org/files/issues/2018-10-05/bef-summary-attributes-3001967-4.patch"
```
Comment (jacine) [inline composer.json:71]: 👏

File: config/install/field.field.apidoc.apidoc.field_categories.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ status: true
 dependencies:
   config:
     - field.storage.apidoc.field_categories
-    - taxonomy.vocabulary.category
+    - taxonomy.vocabulary.api_category
```
Comment (jacine) [inline config/install/field.field.apidoc.apidoc.field_categories.yml:6]: This is better, thanks :)

File: themes/custom/apigee_kickstart/src/components/card/api-card.twig [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{#
+/**
+ * @file
+ * Template for a Card component.
+ */
+#}
+{% set classes = [
+  'card',
+  'api-card'
+]|merge(classes|default([])) %}
```
Comment (jacine) [inline themes/custom/apigee_kickstart/src/components/card/api-card.twig:10]: We should make the other card component support what needs to happen here and use it in a follow up.  This is ok right now.

**Review by jacine — APPROVED** [review]
Thanks for all the help with this @arshad and @minnur.  Looks fantastic!

**Pull request merged**